src/index.ts

```diff
@@ -39,11 +39,16 @@ const Pane = {
     const target = context.dom.createElement("div");
     const store = context.session.store;
 
-    loadExtendedProfile(store, subject).then(() => {
-      render(ProfileView(subject, context), target)
+    loadExtendedProfile(store, subject).then(async () => {
+      const testAAAAA = render(ProfileView(subject, context), target) // @@ await??
```

Comment:
Not sure about this line and the consol log -> was this only for testing purpose?